src/popup/pages/profile.tsx

```diff
@@ -16,200 +31,206 @@ import { optLogOut } from "../../utils/checkAuthentication";
+            <a
+              className="flex flex-col items-center hover:text-orange hover:scale-105"
+              href={`https://${OPEN_SAUCED_INSIGHTS_DOMAIN}/user/${username}`}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              <img
+                alt="User avatar"
+                className="rounded-full w-14 aspect-square p-1 bg-slate-700"
+                src={`https://github.com/${username}.png`}
+              />
+
+              <p className="font-medium">@{username}</p>
+            </a>
+
+            {(user?.linkedin_url || user?.twitter_username) && (
+              <div className="social flex gap-0.5">
+                {user.linkedin_url && (
+                  <a
+                    className="rounded-sm border bg-slate-700 hover:bg-slate-700/50 hover:text-cyan-400 p-1"
+                    href={user.linkedin_url}
+                    rel="noreferrer"
+                    target="_blank"
+                    title={user.linkedin_url}
+                  >
+                    <RiLinkedinFill className="text-lg" />
+                  </a>
+                )}
+
+                {user.twitter_username && (
+                  <a
+                    className="rounded-sm border bg-slate-700 hover:bg-slate-700/50 hover:text-cyan-400 p-1"
+                    href={`https://twitter.com/${user.twitter_username}`}
+                    rel="noreferrer"
+                    target="_blank"
+                    title={`https://twitter.com/${user.twitter_username}`}
+                  >
+                    <RiTwitterFill className="text-lg" />
+                  </a>
+                )}
+              </div>
+            )}
+
+            {user?.bio && <span>{emojify(user.bio)}</span>}
+
+            {user?.blog && (
+              <a
+                className="flex text-orange items-center gap-0.5"
+                href={user.blog}
+                rel="noreferrer"
+                target="_blank"
+              >
+                <RiLinkM />
+
+                {user.blog}
+              </a>
+            )}
+          </div>
+
+          <div className="grid grid-cols-2 text-white bg-osOrange -mx-4 mb-4 p-4 py-8">
+            <div className="flex flex-col items-center justify-center p-2 text-xs">
+              <p>Total Highlights</p>
+
+              <p className="font-medium text-5xl">
+                {userHighlights?.meta.itemCount}
+              </p>
             </div>
-        </div>
-    );
+
+            <div className="flex flex-col items-center justify-center p-2 text-xs">
+              <p>PR Count (30 days)</p>
+
+              <p className="font-medium text-5xl">{userPR?.meta.itemCount}</p>
+            </div>
+
+            <div className="flex flex-col items-center justify-center p-2 text-xs">
+              <p>OSCR</p>
+
+              {user?.bio && (
+                <p className="font-medium text-5xl">{Math.ceil(user.oscr)}</p>
+              )}
```

Comment:
refactor (blocking): The OSCR should render regardless of whether or not the user has a bio. Looking at the way this is currently implemented, `user` can be `undefined` as it gets set in a `useEffect`. That stuff can probably be reworked, but not in this PR.

```suggestion
              {user && (
                <p className="font-medium text-5xl">{Math.ceil(user.oscr)}</p>
              )}
```